subsys/nrf_security/src/drivers/cracen/psa_driver.Kconfig

```diff
@@ -12,2688 +12,821 @@ visible if 0
 config PSA_NEED_CRACEN_CCM_AES
 	bool
 	default y
+	select PSA_ACCEL_CCM_AES_128
+	select PSA_ACCEL_CCM_AES_192
+	select PSA_ACCEL_CCM_AES_256
+	depends on PSA_WANT_AES_KEY_SIZE_128
```

Comment:
I see that for Cryptocell, we document in driver_config.rst:

The :ref:`nrf_security_drivers_cc3xx` is limited to AES key sizes of 128 bits on devices with Arm CryptoCell cc310.

But not for CRACEN.

Did we accidentally copy this dependency from Cryptocell to CRACEN?

Is CRACEN limited to AES128? If so we should update the docs accordingly, if not we should remove this depends (or maybe expand it to depend on all the key sizes).

Reply:
Good catch, started with taking cryptocell as reference and this limitation is not there for CRACEN, so added the other key sizes as well